### PR TITLE
webgl2 change

### PR DESCRIPTION
**Pull request by agirault**
cc: @martinken 
- Use a fixed for loop instead of float variable loop.

### CODE REVIEW

File: Sources/Rendering/OpenGL/RenderWindow/index.js [hunk 1]
```diff
@@ -543,6 +542,10 @@ export function extend(publicAPI, model, initialValues = {}) {
   if (typeof InstallTrigger !== 'undefined') {
     model.defaultToWebgl2 = true;
   }
+  model.defaultToWebgl2 = true;
```
Comment (agirault) [inline Sources/Rendering/OpenGL/RenderWindow/index.js]: @martinken should I remove `defaultToWebgl2` altogether? So we always use webgl2 if available?

File: Sources/Rendering/OpenGL/glsl/vtkVolumeFS2.glsl [hunk 2]
```diff
@@ -165,17 +165,18 @@ void main()
...
     vpos = vpos + vdelta*(0.01 + 0.98*jitter);
     vec4 color = vec4(0.0, 0.0, 0.0, 0.0);
 
+    int count = int(numSteps - 0.2); // end slightly inside
+
     vec3 ijk = vpos * vVCToIJK;
     vdelta = vdelta * vVCToIJK;
-    float i = 0.0;
-    while (i < numSteps - 0.2)
+    for (int i = 0; i < //VTK::MaximumSamplesValue ; ++i)
```
Comment (agirault) [inline Sources/Rendering/OpenGL/glsl/vtkVolumeFS2.glsl]: @martinken how is `i` incrementing with that comment?

File: Sources/Rendering/OpenGL/RenderWindow/index.js (hunk 1 above)
Comment (martinken) [reply Sources/Rendering/OpenGL/RenderWindow/index.js]: I would leave it for now and see how it goes. After a while we can yank it if we find no cases where we need to fall back to webgl 1 even when webgl2 is available.

**Comment by agirault** [comment]
All tests passing

**Pull request merged**